build.gradle.kts

```diff
@@ -30,7 +31,8 @@ buildscript {
 allprojects {
   repositories {
     google()
-    jcenter()
+    jcenter() // Remove, see mapbox-maps-android#92
```

Comment:
Can we remove it already?

Reply:
nope, some upstream dependencies are still not hosted yet on mavencentral..